lib/mongify/database/sql_connection.rb

```diff
@@ -76,21 +77,22 @@ def columns_for(table_name)
       end
 
       # Returns an array with hash values of all the records in a given table
-      def select_rows(table_name, &block)
+      def select_rows(table_name, is_sync, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.